### ISSUE TITLE
NTR: Renal biopsy: cystine crystals (Lab: pathology: biopsies & morphology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9133]
New term request Renal biopsy: cystine crystals (elthat_100520034404): Kidney

**Comment by pnrobinson** [comment]
@MickeySegal
is that this pathology? https://www.kidney-international.org/article/S0085-2538(15)51432-4/fulltext

@avirosenberg -- could you help define this term? Should we specify which part of the nephron?

**Comment by MickeySegal** [comment]
https://www.ncbi.nlm.nih.gov/books/NBK1400/ describes "Electron microscopy of the glomeruli reveals fusion of foot processes, thickening of Bowman's membrane, and localization of cystine crystals in interstitial cells".  In the article you cite it sounds like the crystals are deposited in many places in the kidney, so the more general term might be best.

**Comment by pnrobinson** [comment]
@MickeySegal 
I think this is the most widely used terminology:

Adding new term: 
- ID: HP:6001031
- label: Cystine nephrolithiasis
- parents: [Nephrolithiasis (HP:0000787)](https://hpo.jax.org/app/browse/term/HP:0000787)
- pmid: PMID:25383320|PMID:32494553
- def: Deposition of cystine crystals in kidney tissue. Cystine crystals display a pathognomic hexagonal shape.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9133
- synonyms:
